python/ray/tune/trainable/trainable.py

```diff
@@ -420,6 +420,14 @@ def train(self):
 
         return result
 
+    @property
+    def _checkpoint_save_restore_delete_retry_args(self) -> dict:
+        return dict(
+            num_retries=int(os.getenv("TUNE_CHECKPOINT_CLOUD_RETRY_NUM", "3")),
+            sleep_time=float(os.getenv("TUNE_CHECKPOINT_CLOUD_RETRY_WAIT_TIME_S", "1")),
+            timeout=self.sync_timeout,
+        )
```

Comment:
The function name is a bit too long, maybe just `_sync_retry_args`?

Actually, I think we should just store these on init in `self._sync_num_retries` and `self._sync_sleep_time`.